SygicMapsKit/Classes/BrowseMap/SYMKBrowserMapView.swift

```diff
@@ -50,4 +50,11 @@ class SYMKBrowserMapView: UIView {
         compass.topAnchor.constraint(equalTo: safeTopAnchor, constant: sideMargin).isActive = true
     }
     
+    private func setupRecenter() {
+        addSubview(recenter)
+        bringSubview(toFront: recenter)
+        recenter.translatesAutoresizingMaskIntoConstraints = false
+        recenter.leadingAnchor.constraint(equalTo: safeLeadingAnchor, constant: 32).isActive = true
+        recenter.bottomAnchor.constraint(equalTo: safeBottomAnchor, constant: -16).isActive = true
```

Comment:
Preco je toto raz 32 a 16? Tipujem, ze na Xku to vyzera takto lepsie? lebo na nonX iphonoch to vyzera celkom zle. Pre normal iphony by to mohlo byt 16, teda hore vytvorena konstanta `sideMargin`, akurat neviem ako to vyzera na X

Reply:
done